dlp/src/test/java/com/example/dlp/RiskAnalysisIT.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dlp;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.*;
+
+@RunWith(JUnit4.class)
+public class RiskAnalysisIT {
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out); // TODO(b/64541432) DLP currently doesn't support GOOGLE DEFAULT AUTH
+    assertNotNull(System.getenv("GOOGLE_APPLICATION_CREDENTIALS"));
+    assertNotNull(System.getenv("DLP_DEID_WRAPPED_KEY"));
```

Comment:
if you can hard-code these variables, that would be ideal.
Else .travis.yaml will need to be updated so that these tests are successful on `java-docs-samples-testing`.

Reply:
In that case, agree on not hard-coding. @lesv might be able to recommend the best approach.

Reply:
Ace - can you just add how to do this to the README.  I'm a fan of getting this stuff from Datastore, but in this case I'm going to go w/ Ace's current implementation.

We should also mention it in the root README as well - once we write it.

Reply:
Done.